search/src/Components/wrapper/reactive_search.js

```diff
@@ -627,6 +618,15 @@ class ReactiveSearchComponent extends Component {
                 </div>
                 <div style={{ position: "absolute", left: "20rem", paddingRight: "1rem" }}>
                     <SelectedFilters clearAllLabel="Clear filters" />
+                    <MultiDropdownList
+                        componentId="dataset_name_filter"
+                        dataField="dataset_name.keyword"
+                        title="Dataset Name"
```

Comment:
```suggestion
                        title="Datasets"
```

(I did think about making this dynamic, e.g. `title={{numberOfDatasets} + " Datasets"}`, but not sure of how to extend the render hook on MultiDropdownList)